README.md

```diff
@@ -49,6 +51,13 @@ Supported Top-level Domains
 * im
 * uk
 * co.uk
+* tech
+* co
+* digital
+
+Supported Whois Servers
+
+* whois.crazydomains.com
```

Comment:
I see that `check_domain_by_whois()` has two supported WHOIS servers, could you please add this information also here?

Reply:
Oh right. done